src/scenes/ProgressReports/EditForm/Steps/ProgressStep/ProgressStep.tsx

```diff
@@ -0,0 +1,239 @@
+                (TENTATIVE COPY) Please upload the PnP for this reporting
+                period. The progress data will populate the charts below.
+              </Typography>
+            </Grid>
+          )}
+          <Grid item md={6} justifyItems="end">
+            <ProgressReportCard
+              progressReport={report}
+              disableIcon
+              onUpload={({ files }) => setUploading(files)}
+            />
+          </Grid>
+        </Grid>
+      </Grid>
+
+      {categories.map((category) => (
+        <Box key={category} sx={{ mb: 4 }}>
+          <EditableProductTable
+            products={forTable[category]}
+            category={category}
+            update={update}
+            report={report}
+            tab={tab}
+            setTab={setTab}
+          />
+        </Box>
+      ))}
+
+      <UpdatePeriodicReportDialog
+        {...dialogState}
+        report={{ ...report, reportFile: upload }}
+        editFields={[
+          'receivedDate',
+          ...(upload && upload.length > 0 ? ['reportFile' as const] : []),
+        ]}
+      />
+    </div>
+  );
+};
+
+const EditableProductTable = ({
+  category,
+  products = [],
+  update,
+  report,
+  tab,
+  setTab,
+}: {
+  category: string;
+  products?: ProgressReportProgressFragment[];
+  update: any;
+  report: ProgressReportEditFragment;
+  tab: typeof variants[number];
+  setTab: (tab: typeof variants[number]) => void;
+}) => (
+  <ProductTable
+    category={category}
+    products={products}
+    pagination
+    header={() => (
+      <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <VariantsToggle
+          setTab={(value) => setTab(variants[value] ?? variants[0])}
+          tab={variants.indexOf(tab)}
+        />
+      </div>
+    )}
+    editMode="row"
+    onRowEditStop={(fields) => {
+      void update({
+        variables: {
+          input: {
+            productId: fields.id.toString(),
+            reportId: report.id,
+            variant: tab,
+            steps: [
+              ...fields.row.data.steps.map((step: StepProgressFragment) => ({
+                completed: parseFloat(fields.row[step.step]),
+                percentDone: parseFloat(fields.row[step.step]),
+                step: step.step,
+              })),
+            ],
+          },
+        },
+      });
+    }}
+  />
+);
+
+const VariantsToggle = ({
+  tab,
+  setTab,
+}: {
+  tab: number;
+  setTab: (tab: number) => void;
+}) => {
+  return (
+    <ToggleButtonGroup
+      value={tab}
+      exclusive
+      onChange={(_e, value) => {
+        if (value !== null) {
+          setTab(value);
+        }
+      }}
+      color="secondary"
+      sx={{ mt: 1, ml: 1 }}
+    >
+      <ToggleButton
+        value={0}
+        sx={[ToggleButtonSx('FieldPartner'), tab !== 0 && transparentBgSx]}
+      >
+        <RoleIcon
+          // eslint-disable-next-line jsx-a11y/aria-role
+          role="FieldPartner"
```

Comment:
I think it would probably be good to just rename this to something else so we don't have to avoid the conflicting a11y prop. What about `value`?